ceph_deploy/hosts/fedora/__init__.py

```diff
@@ -19,9 +18,9 @@ def choose_init(module):
     """
 
     if not module.conn.remote_module.path_exists("/usr/lib/systemd/system/ceph.target"):
-        return 'sysvinit'
+        return init_systems.SysV(module)
```

Comment:
Am I correct in thinking this is now returning a class?

This would also be a very good time to "inject" the service mappings, for fedora rather than have them hard coded in "init_systems"

If so the comments for all OS need changing.